<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Launching Browser]]>
    </line>
    <line>
      <![CDATA[Player Logged in Successfully,SignIn TestCase Passed !!!!]]>
    </line>
    <line>
      <![CDATA[Closing the browser]]>
    </line>
    <line>
      <![CDATA[Launching Browser]]>
    </line>
    <line>
      <![CDATA[Mines Game Played Successfully]]>
    </line>
    <line>
      <![CDATA[Closing the browser]]>
    </line>
    <line>
      <![CDATA[Launching Browser]]>
    </line>
    <line>
      <![CDATA[Closing the browser]]>
    </line>
  </reporter-output>
  <suite started-at="2025-07-06T20:40:55 IST" name="Suite" finished-at="2025-07-06T20:42:29 IST" duration-ms="93956">
    <groups>
    </groups>
    <test started-at="2025-07-06T20:40:55 IST" name="Test" finished-at="2025-07-06T20:42:29 IST" duration-ms="93956">
      <class name="test.Test1">
        <test-method is-config="true" signature="launchCryptocrash()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:40:55 IST" name="launchCryptocrash" finished-at="2025-07-06T20:41:07 IST" duration-ms="11951" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Launching Browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchCryptocrash -->
        <test-method is-config="true" signature="intiPageObjects()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:07 IST" name="intiPageObjects" finished-at="2025-07-06T20:41:08 IST" duration-ms="135" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intiPageObjects -->
        <test-method signature="signIN()[pri:1, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:08 IST" name="signIN" finished-at="2025-07-06T20:41:10 IST" duration-ms="2816" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Player Logged in Successfully,SignIn TestCase Passed !!!!]]>
            </line>
          </reporter-output>
        </test-method> <!-- signIN -->
        <test-method is-config="true" signature="closeCryptocrash()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:10 IST" name="closeCryptocrash" finished-at="2025-07-06T20:41:12 IST" duration-ms="1430" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Closing the browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeCryptocrash -->
        <test-method is-config="true" signature="launchCryptocrash()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:12 IST" name="launchCryptocrash" finished-at="2025-07-06T20:41:19 IST" duration-ms="7575" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Launching Browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchCryptocrash -->
        <test-method is-config="true" signature="intiPageObjects()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:19 IST" name="intiPageObjects" finished-at="2025-07-06T20:41:19 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intiPageObjects -->
        <test-method signature="play_MinesManualGame()[pri:3, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:19 IST" name="play_MinesManualGame" finished-at="2025-07-06T20:41:44 IST" duration-ms="25086" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Mismatch at column 2 expected [48] but found [48
User 445]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Mismatch at column 2 expected [48] but found [48
User 445]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at test.Test1.play_MinesManualGame(Test1.java:112)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Mines Game Played Successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- play_MinesManualGame -->
        <test-method is-config="true" signature="closeCryptocrash()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:44 IST" name="closeCryptocrash" finished-at="2025-07-06T20:41:46 IST" duration-ms="1416" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Closing the browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeCryptocrash -->
        <test-method is-config="true" signature="launchCryptocrash()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:46 IST" name="launchCryptocrash" finished-at="2025-07-06T20:41:54 IST" duration-ms="8137" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Launching Browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchCryptocrash -->
        <test-method is-config="true" signature="intiPageObjects()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:54 IST" name="intiPageObjects" finished-at="2025-07-06T20:41:54 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intiPageObjects -->
        <test-method signature="play_MinesAutoGame()[pri:4, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:41:54 IST" name="play_MinesAutoGame" finished-at="2025-07-06T20:42:28 IST" duration-ms="33869" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Total Number of Rounds didn't Matched TC Failed!!!! expected [Total completed rounds: 10] but found [Total completed rounds: 7]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Total Number of Rounds didn't Matched TC Failed!!!! expected [Total completed rounds: 10] but found [Total completed rounds: 7]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at test.Test1.play_MinesAutoGame(Test1.java:139)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- play_MinesAutoGame -->
        <test-method is-config="true" signature="closeCryptocrash()[pri:0, instance:test.Test1@6e171cd7]" started-at="2025-07-06T20:42:28 IST" name="closeCryptocrash" finished-at="2025-07-06T20:42:29 IST" duration-ms="1448" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Closing the browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeCryptocrash -->
      </class> <!-- test.Test1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
